SeleniumPracticalExercise/PageObjects/AddingEmployee.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OpenQA.Selenium;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class AddingEmployee : BasePageLocal
+    {
+        private readonly By PIMLink = By.XPath("//span[text()='PIM']"); 
```

Comment:
All your locators should be in the format `_<element name>Locator`, e.g. `_pimLinkLocator`. Please change them everywhere to be consistent.

Reply:
renamed locators to have the word Locator in it